src/main/scala/vsys/settings/WebhookEventSettings.scala

```diff
@@ -0,0 +1,98 @@
+package vsys.settings
+
+trait WebhookEventSettings{
+  val typeId: Int 
+  val typeDescription: String
+}
+
+case class BlockAppendedEventSettings(
+  withTxs: Boolean, 
```

Comment:
with transaction or with non-minting transaction here?

Reply:
The withTxs means as a block appended, the system will send an event notification with all the transaction in that block. We can also add one more field withNonMintingTxs to be more specific.

Reply:
i see